monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -369,6 +369,23 @@ link:#prometheusrestrictedconfig[PrometheusRestrictedConfig]
 
 |===
 
+== TelemeterClientConfig
+
+=== Description
+
+The `TelemeterClientConfig` resource defines settings for the `telemeter-client` component.
+
+Appears in: link:#clustermonitoringconfiguration[ClusterMonitoringConfiguration]
+
+[options="header"]
+|===
+| Property | Type | Description 
+|nodeSelector|map[string]string|Defines the nodes on which the pods are scheduled.
+
+|tolerations|[]v1.Toleration|Defines tolerations for the pods.
+
+|===
+
```

Comment:
@JoaoBraveCoding 
from https://github.com/openshift/cluster-monitoring-operator/blob/release-4.12/pkg/manifests/config.go#L175-L182
we can configure all the parameters expect ClusterID, but only nodeSelector/tolerations are mentioned here, is there any reason we don't want to list other parameters in the doc?

Reply:
Yes because users aren't supposed to modify these parameters. I assume that they were added to enabling testing with the Telemetry server. But the `enabled` field for example doesn't appear in the OCP docs as a way to disable telemetry.

Reply:
@juzhao Please review Simon's comment and respond. If you agree, please provide an lgtm for this PR. Thanks!